core/dbt/contracts/graph/nodes.py

```diff
@@ -144,6 +146,8 @@ class ColumnInfo(AdditionalPropertiesMixin, ExtensibleDbtClassMixin, Replaceable
     description: str = ""
     meta: Dict[str, Any] = field(default_factory=dict)
     data_type: Optional[str] = None
+    constraints: Optional[List[str]] = None
+    check: Optional[str] = None
```

Comment:
`check` is pretty vague. What are we checking? The constraints? The data_type? A more specific name might be useful here.

Reply:
`check` is a normal term for checking arbitrary things like `check (id > 0)`, so for SQL table config context, it's not vague.

https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-CHECK-CONSTRAINTS

Let me know if you referring to a different angle of vague.

Reply:
Same language used by Databricks: https://docs.databricks.com/tables/constraints.html

Reply:
+1 that this is pretty vague, maybe `constraints_check`? In the case you shared the user already has context that the check relates to constraints. In our case there isn't an obvious link between check and constraints

Reply:
I can update this to `constraints_check`

Reply:
All `check` configs have been updated to `constraints_check`